scripts/run_mypy_checks.py

```diff
@@ -751,6 +750,12 @@
     help='If true, skips installing dependencies. The default value is false.',
     action='store_true')
 
+_PARSER.add_argument(
+    '--install-globally',
+    help='If true, installs mypy and its requirements globally.'
+    ' By default, they are installed to ' + MYPY_TOOLS_DIR,
```

Comment:
I still find the "If true" part confusing since user doesn't set this flag to be true or false--they either pass it or don't

Reply:
```
help='optional; if specified, installs mypy and its requirements globally.'
    ' By default, they are installed to  %s' % ...
```

Reply:
Changed. Thanks!